applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -313,6 +352,21 @@ set_system_macros(Context) ->
             Context
     end.
 
+post(Context, ?CUSTOMER_UPDATE, ?MESSAGE) ->
```

Comment:
No, post/3 spec is a bit upstairs, introduced with post/2.
https://github.com/onnet/kazoo/blob/e0b1365578444a14e2788ca512cdc602a299f78a/applications/crossbar/src/modules/cb_notifications.erl#L319

Will move it closer to not to be confused